### PR TITLE
sign-in sign-up pages mobile versions

### PR DESCRIPTION
**Pull request by Llavrov**

### CODE REVIEW

File: src/components/pages/signIn/index.tsx [hunk 1]
```diff
@@ -71,34 +71,43 @@ export default function SignInPage(): ReactElement {
...
+                    <div className="flex justify-between w-full max-w-[400px]">
+                        <HStack className="items-center">
                             <PECheckbox checked={staySignedIn} onCheckedChange={setStaySignedIn} />
                             <p>Remember me</p>
                         </HStack>
-                        <Spacer />
-                        <div style={{ flex: 1 }}></div>
+                        <div className="lg:hidden">
+                            <Spacer />
+                        </div>
```
Comment (CemYil03) [inline src/components/pages/signIn/index.tsx:81]: Please don't use css hidden, instead make it TypeScript driven

Comment (Llavrov) [reply src/components/pages/signIn/index.tsx:81]: oh, okay, got you

Comment (Llavrov) [reply src/components/pages/signIn/index.tsx:81]: But could you please write an example, how to make it hidden by ts instead css

File: src/components/header/PEHeader.tsx [hunk 2]
```diff
@@ -3,29 +3,32 @@ import useTranslation from 'next-translate/useTranslation';
...
 
-                <Link href={'/sign-in'}>
-                    <Button style={{ color: 'rgba(31, 31, 31, 0.8)' }}>{t('sign-in')}</Button>
-                </Link>
-            </HStack>
+                    <Link href={'/sign-in'}>
+                        <Button style={{ color: 'rgba(31, 31, 31, 0.8)' }}>{t('sign-in')}</Button>
+                    </Link>
+                </div>
+            ) : null}
```
Comment (CemYil03) [inline src/components/header/PEHeader.tsx:31]: So for mobile we would not show anything now?

File: src/components/header/PEHeaderMobile.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    const [openMenu, setOpenMenu] = useState(false);
+    const [sticky, setSticky] = useState(false);
+    const responsive = useResponsive();
+
+    useEffect(() => {
+        document.addEventListener('scroll', () => {
+            if (window.pageYOffset > 80) setSticky(true);
+            else if (window.pageYOffset <= 160) setSticky(false);
+        });
+    }, []);
```
Comment (CemYil03) [inline src/components/header/PEHeaderMobile.tsx:21]: It is not even sure if we really want it to be sticky, should check with Natalia and Daniel.

File: src/components/pages/chefProfile/index.tsx [hunk 4]
```diff
@@ -11,6 +12,7 @@ export default function ChefProfilePage(): ReactElement {
     return (
         <VStack className="w-full">
             <PEHeader />
+            <PEHeaderMobile />
```
Comment (CemYil03) [inline src/components/pages/chefProfile/index.tsx:15]: Here is the correct place where we check if we are on mobile or desktop, not within the components themselves:

```suggestion
            {isDesktop && <PEHeader />}
            {isMobile && <PEHeaderMobile />}
```

Comment (CemYil03) [reply src/components/pages/chefProfile/index.tsx:15]: Same thing for every other page

File: src/components/pages/signIn/index.tsx [hunk 5]
```diff
@@ -71,49 +73,58 @@ export default function SignInPage(): ReactElement {
...
-                        <HStack className={'items-center'}>
+                    <div className="flex justify-between w-full max-w-[400px]">
+                        <HStack className="items-center">
                             <PECheckbox checked={staySignedIn} onCheckedChange={setStaySignedIn} />
                             <p>Remember me</p>
                         </HStack>
-                        <Spacer />
-                        <div style={{ flex: 1 }}></div>
+                        {responsive.isDesktop ? <Spacer /> : null}
+                        {responsive.isDesktop ? <div style={{ flex: 1 }}></div> : null}
```
Comment (CemYil03) [inline src/components/pages/signIn/index.tsx:82]: In most cases if you use the ternary operator defaulting to `null`, consider just using the `&&` operator.
Also replace line 82 with a `<Spacer />`. This is exactly what it does, having `flex: 1`

File: src/components/pages/signUp/index.tsx [hunk 6]
```diff
@@ -156,28 +160,35 @@ export default function SignUpPage(): ReactElement {
...
             </VStack>
-
-            <VStack className={'p-5 w-full'} style={{ flex: 1 }}>
-                <VStack
-                    className={'w-full rounded-tl-15'}
-                    style={{ flex: 1, backgroundImage: 'url(/picture-1.png)', backgroundPosition: 'center', backgroundSize: 'cover' }}
-                >
-                    <Spacer />
-                    <VStack
+            {responsive.isDesktop ? (
```
Comment (CemYil03) [inline src/components/pages/signUp/index.tsx:163]: Switch to `&&` here as well

**Review by CemYil03 — CHANGES_REQUESTED** [review]

File: src/components/header/PEHeader.tsx (hunk 2 above)
Comment (Llavrov) [reply src/components/header/PEHeader.tsx:31]: oh, yes, is good idea just to show MobileHeader

**Pull request merged**